# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: tools/lint/clippy.yml
# Commit: ef5dc3e04e5f
# Full Hash: ef5dc3e04e5f271eea0636ab3a495e95cc912f1d
# Author: Dzmitry Malyshau <dmalyshau@mozilla.com>
# Date: 2021-09-04 09:40:24
# Regressor Bug: 1726626
# File Overlap Count: 3
# Description:
#   Bug 1726626 - Move gfx/wgpu into a 3rd party dependency r=jgilbert,bholley
#   
#   This update makes wgpu a vendored dependency instead of having it in gfx/wgpu.
#   
#   ## Notes
# ==============================================================================

diff --git a/tools/lint/clippy.yml b/tools/lint/clippy.yml
--- a/tools/lint/clippy.yml
+++ b/tools/lint/clippy.yml
@@ -51,17 +51,16 @@ clippy:
         # derive_hash_xor_eq
         - gfx/wr/
         - gfx/wr/webrender/
         - gfx/wr/examples/
         # windows-only
         - gfx/wr/example-compositor/compositor-windows/
         - gfx/wr/webrender_api/
         - gfx/wr/wrench/
-        - gfx/wgpu/wgpu-core/
         - gfx/wgpu_bindings/
         # not_unsafe_ptr_arg_deref
         - modules/libpref/parser/
         - tools/profiler/rust-helper/
         - toolkit/library/rust/shared/
         - toolkit/library/gtest/rust/
         # not_unsafe_ptr_arg_deref
         - remote/